src/pages/User/components/PullRequests/index.js

```diff
@@ -71,7 +71,7 @@ export default class PullRequests extends Component {
       const [data, userDetail] = await Promise.all(allResponses);
       const count = this.counterOtherRepos(data, userDetail);
       this.setState({
-        data,
+        data: this.getValidPullRequest(data),
```

Comment:
It returns a list of pull requests (plural) - thus its name should reflect that too. 
```suggestion
        data: this.getValidPullRequests(data),
```